### PR TITLE
[DF] Log begin/end of event loop and processing of each entry range

### PR DESCRIPTION
**Pull request by eguiraud**
Now using the new and improved RLogger!

### CODE REVIEW

**Comment by eguiraud** [comment]
@phsft-bot build

**Comment by eguiraud** [comment]
@phsft-bot build

**Comment by eguiraud** [comment]
@reviewers please take a look!

File: tree/dataframe/src/RLoopManager.cxx [hunk 1]
```diff
@@ -226,6 +232,54 @@ struct MaxTreeSizeRAII {
...
+struct DatasetLogInfo {
+   std::string fDataSet;
+   ULong64_t fRangeStart;
+   ULong64_t fRangeEnd;
+};
+
+std::string LogRangeProcessing(const DatasetLogInfo &info)
+{
+   std::stringstream msg;
+   msg << "Processing " << info.fDataSet << ": entry range [" << info.fRangeStart << "," << info.fRangeEnd - 1
```
Comment (etejedor) [inline tree/dataframe/src/RLoopManager.cxx:244]: Do you think that including info about the slot index could be useful here as well?

File: tree/dataframe/src/RLoopManager.cxx [hunk 2]
```diff
@@ -556,13 +618,21 @@ void RLoopManager::CleanUpTask(unsigned int slot)
...
 
    const std::string code = std::move(GetCodeToJit());
-   if (code.empty())
+   if (code.empty()) {
+      R__LOG_INFO(RDFLogChannel()) << "Nothing to jit and execute.";
       return;
+   }
 
+   TStopwatch s;
+   s.Start();
```
Comment (etejedor) [inline tree/dataframe/src/RLoopManager.cxx:631]: So you are going to measure the jitting time always, not just when you are logging? Does this happen once per event loop?

**Review by etejedor — APPROVED** [review]
LGTM! Made a couple of comments.

File: tree/dataframe/src/RLoopManager.cxx (hunk 1 above)
Comment (eguiraud) [reply tree/dataframe/src/RLoopManager.cxx:244]: why not :D

File: tree/dataframe/src/RLoopManager.cxx (hunk 2 above)
Comment (eguiraud) [reply tree/dataframe/src/RLoopManager.cxx:631]: yes, but `Start` and `Stop` are basically just taking a timestamp, they are cheap (esp. compared to typical jitting or event loop times)

File: tree/dataframe/src/RLoopManager.cxx (hunk 1 above)
Comment (eguiraud) [reply tree/dataframe/src/RLoopManager.cxx:245]: done

**Pull request merged**